setup.php

```diff
@@ -1478,14 +1398,33 @@ function cloneClicked() {
                             echo "<br><FONT COLOR='green'>Ensuring following directories have proper permissions...</FONT><br>\n";
                             $errorWritable = 0;
                             foreach ($writableDirList as $tempDir) {
+                                if($tempDir==$docsDirectory){
```

Comment:
This is definitely the right placement for this. I plan to do a some research tomorrow to find a mechanism to recursively collect directories.

Reply:
Wow, this ended up being a bit more complicated than I thought it would be. While I was figuring this out I ended basically doing the coding for this improvement here:
https://github.com/openemr/openemr/pull/2273
(I listed you as a coauthor on the commit since your solution was a solid starting point)